fluent-dom/src/localization.js

```diff
@@ -24,6 +24,16 @@ export default class Localization {
       new CachedAsyncIterable(this.generateMessages(this.resourceIds));
   }
 
+  addResourceIds(resourceIds) {
+    this.resourceIds.push(...resourceIds);
+    this.onLanguageChange();
```

Comment:
Now that `onLanguageChange` is called also for resource ids changes, should we rename it to something more general?

Reply:
Yeah, we probably want to rename it. Any preference for the new name? :)

Reply:
Not really. Is `onChange` too generic? :)